packages/algob/sample-project/common/.gitignore

```diff
@@ -0,0 +1,6 @@
+/node_data
+.dockerignore
+artifacts/
+/algob.config-local.js
```

Comment:
Why use add /algob.config-local.js here?

Reply:
I think it use to ignore the local file in the computer?